homeassistant/components/mqtt/climate.py

```diff
@@ -455,7 +480,7 @@ def _setup_from_config(self, config):
             for key, template in value_templates.items()
         }
 
-        command_templates = {}
+        command_templates: dict[str, Callable[..., PublishPayloadType]] = {}
         for key in COMMAND_TEMPLATE_KEYS:
             command_templates[key] = MqttCommandTemplate(
```

Comment:
```suggestion
            self._command_templates[key] = MqttCommandTemplate(
```

And need to remove line 489 below: `self._command_templates = command_templates`